pipeline/relation_embedder/relation_embedder/src/main/scala/weco/pipeline/relation_embedder/models/ArchiveRelationsCache.scala

```diff
@@ -33,43 +34,6 @@ class ArchiveRelationsCache(works: Map[String, RelationWork]) extends Logging {
 
   private val paths: PathCollection = PathCollection(works.keySet)
 
-  /** Find the availabilities of this Work.
-    *
-    * The availabilities of an archive's Relations are the union of all the
-    * availabilities of its descendents, as well as its own.
-    *
-    * Note that this only covers *known* descendents.  e.g. if the works have
-    * paths (A, A/B/1), then A will not inherit availabilities from A/B/1 --
-    * the intermediate path A/B is missing.
-    *
-    * This preserves the original behaviour of the relation embedder, but it's
-    * not clear if it's intentional -- it was a side effect of the implementation,
-    * not something that was explicitly tested.
```

Comment:
Hahaha, I remember writing this comment and wondering if we actually wanted this behaviour

Reply:
Ha, yes, it was intentional but done kind of last-minute...